spec/services/monthly_district_data_service_spec.rb

```diff
@@ -140,18 +164,20 @@ def find_in_csv(csv_data, row_index, column_name)
       expect(find_in_csv(csv, row_index, "Patients with a visit but no BP taken")).to eq("1")
       expect(find_in_csv(csv, row_index, "Patients with a visit but no BP taken")).to eq("1")
       expect(find_in_csv(csv, row_index, "Patients under care as of #{period.end.strftime("%e-%b-%Y")}")).to eq("1")
-      expect(csv[row_index][28..30].uniq).to eq([nil])
+      expect(csv[row_index][31]).to eq("1")
+      expect(csv[row_index][36]).to eq("1")
+      expect(csv[row_index][40..42].uniq).to eq([nil])
     end
 
     it "scopes the report to the provided period" do
-      old_period = Period.month("July 2018")
+      old_period = Period.current
       result = described_class.new(region, old_period).report
       csv = CSV.parse(result)
-      column_headers = csv[2]
+      column_headers = csv[3]
       first_month_index = 11
       last_month_index = 16
-      expect(column_headers[first_month_index]).to eq("Feb-2018")
-      expect(column_headers[last_month_index]).to eq("Jul-2018")
+      expect(column_headers[first_month_index]).to eq(Period.month(5.month.ago).to_s)
+      expect(column_headers[last_month_index]).to eq(Period.current.to_s)
     end
```

Comment:
Let's add a spec that describes the sections, sub-sections and headers.

Reply:
Addressed in [150d794](https://github.com/simpledotorg/simple-server/pull/3349/commits/150d794cf310e13991b9a136ddb49ea1e4d0e9b4)